### PR TITLE
feat: Uninstall brew & others

### PR DESCRIPTION
**Pull request by armenzg**
This change ensures that we uninstall brew and `sentry-cli` in order to test out that execution path.

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -87,7 +87,7 @@ https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/
 
 get_code_root_path() {
   if [ -z "$CODE_ROOT" ]; then
-    read -rp "--> Enter the absolute path to your code [$HOME/code]: " CODE_ROOT
+    read -rp "--> Enter absolute path where to clone source code [$HOME/code]: " CODE_ROOT
```
Comment (armenzg) [inline bootstrap.sh:90]: @wedamija Is this better wording?

File: .github/workflows/development-environment.yml [hunk 2]
```diff
@@ -38,6 +38,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # This will ensure that we test the sentry-cli and brew installation step
+      - name: Uninstall brew & others
+        run: |
+          ls -l /usr/local/bin
```
Comment (armenzg) [inline .github/workflows/development-environment.yml:44]:
```suggestion
```

File: uninstall.sh [hunk 3]
```diff
@@ -5,11 +5,13 @@
 # This script tries to remove as much as reasonable to allow for re-testing
 # bootstrap.sh multiple times on the same host
 
-# Remove Sentry and dependencies
-rm -rf ~/.sentry
-rm -rf ~/code/sentry/{.venv,node_modules}
-rm -rf ~/code/getsentry/{.venv,node_modules}
-[ -f ~/.zprofile ] && rm ~/.zprofile
+remove_other() {
```
Comment (armenzg) [inline uninstall.sh:8]: Just indentation.

File: uninstall.sh [hunk 4]
```diff
@@ -19,11 +21,13 @@ remove_brew_setup() {
     # Execute the official uninstall command
     /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"
     # This is not removed by the script.
-    # On Intel machines, I would not dare removing /usr/local
-    [ -f /opt/homebrew ] && rm -rf /opt/homebrew
+    [ -d /opt/homebrew ] && sudo rm -rf /opt/homebrew
+    [ -d /usr/local/bin ] && sudo rm -rf /usr/local/bin
```
Comment (armenzg) [inline uninstall.sh:25]: This is the actual change.

File: .github/workflows/development-environment.yml [hunk 5]
```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   fast-bootstrap:
     runs-on: macos-11
-    timeout-minutes: 45
```
Comment (armenzg) [inline .github/workflows/development-environment.yml:11]: It doesn't take longer than 10-15 minutes.

File: bootstrap.sh (hunk 1 above)
Comment (billyvg) [reply bootstrap.sh:90]: maybe "Enter absolute path where we'll clone source code"?

Comment (armenzg) [inline bootstrap.sh:90]:
```suggestion
    read -rp "--> Enter absolute path where we'll clone source code [$HOME/code]: " CODE_ROOT
```

**Comment by armenzg** [comment]
r? I committed your wording.

**Review by billyvg — APPROVED** [review]

**Pull request merged**